### PR TITLE
WINC-958: Collect WICD logs from Windows nodes

### PR DESCRIPTION
**Pull request by saifshaikh48**
This PR adds the Windows Instance Config Daemon (WICD) 
to the list of service log files collected from Windows nodes.

### CODE REVIEW

**Comment by aravindhp** [comment]
@davemulford please review

File: collection-scripts/gather_windows_node_logs [hunk 1]
```diff
@@ -4,8 +4,10 @@ WINDOWS_NODE_LOGS=$BASE_COLLECTION_PATH/host_service_logs/windows
 
 # Logfile list
 LOGS=(kube-proxy/kube-proxy.exe.INFO kube-proxy/kube-proxy.exe.ERROR kube-proxy/kube-proxy.exe.WARNING)
-LOGS+=(hybrid-overlay/hybrid-overlay.log kubelet/kubelet.log)
+LOGS+=(hybrid-overlay/hybrid-overlay.log)
+LOGS+=(kubelet/kubelet.log)
 LOGS+=(containerd/containerd.log)
+LOGS+=(wicd/windows-instance-config-daemon.exe.INFO wicd/windows-instance-config-daemon.exe.ERROR wicd/windows-instance-config-daemon.exe.WARNING)
```
Comment (sebsoto) [inline collection-scripts/gather_windows_node_logs:10]: Should we just collect /var/log?

Comment (aravindhp) [reply collection-scripts/gather_windows_node_logs:10]: Wouldn't we pick up container logs also if we do that?

Comment (sebsoto) [reply collection-scripts/gather_windows_node_logs:10]: Okay I'm fine with how it is currently then

**Pull request merged**